### ISSUE TITLE
[BUG] rossmann nvt+tf inference on triton crashes after receiving a query

### ISSUE THREAD

**Issue by oyilmaz-nvidia** [NVIDIA/NVTabular#961]
**Describe the bug**
After starting the triton server for rossmann ensemble model (nvt + tf) with the python backend, if you send a query, it crashes the triton and gives a segfault.

Below is the gdb output:

```
Thread 177 "tritonserver" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7fee57fff000 (LWP 6844)]
0x00007ff7ad8aa44f in triton::backend::python::Tensor::Tensor(triton::backend::python::Tensor const&) () from /opt/tritonserver/backends/python/libtriton_python.so
```

**Steps/Code to reproduce bug**
If you run the integration tests in this PR https://github.com/NVIDIA/NVTabular/pull/848 with order defined in this PR https://github.com/NVIDIA/NVTabular/pull/863#event-5028551967, with the main branch pulled down in the containers, you can reproduce the bug.

**Comment by oyilmaz-nvidia** [comment]
Ran the same integration test with NVTabular C++ backend got the following error (btw, C++ backend did't crash the triton :) );

```E       tritonclient.utils.InferenceServerException: [StatusCode.INVALID_ARGUMENT] in ensemble 'rossmann', unexpected deadlock, at least one output is not set while no more ensemble steps can be made

/usr/local/lib/python3.8/dist-packages/tritonclient/grpc/__init__.py:61: InferenceServerException```

Looks like, we are missing one of the outputs here.

**Comment by oyilmaz-nvidia** [comment]
After printing the transformed data columns in model.py, noticed that not all of the required outputs are being returned. And, this is the reason why we get that error. Below is the list of the columns that are included in the transformed data;

```
Store
DayOfWeek
Year
Month
Day
StateHoliday
CompetitionMonthsOpen
Promo2Weeks
StoreType
Assortment
PromoInterval
CompetitionOpenSinceYear
Promo2SinceYear
State
Week
Events
Promo_fw
Promo_bw
StateHoliday_fw
StateHoliday_bw
SchoolHoliday_fw
SchoolHoliday_bw```

And we are missing many columns here.

**Comment by benfred** [comment]
I have a fix in #972 -  I didn't see this when I was testing myself because this doesn't happen with #926
